### PR TITLE
[Feature][Connector] add console limit and batch flink fake source

### LINKED ISSUE #1506: [Feature][Connector] Flink console sink need lines limit
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Same with spark console connector, flink console connector also need lines limit, avoid too much result print in console

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Hisoka-X**
## Purpose of this pull request
1. add flink console sink lines limit
2. add flink fake source in batch mode
This close #1506 

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If any new Jar binary package adding in you PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/developement/NewLicenseGuide.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-fake/src/main/java/org/apache/seatunnel/flink/source/FakeSource.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    }
+
+    @Override
+    public DataSet<Row> getData(FlinkEnvironment env) {
+        Random random = new Random();
+        return env.getBatchTableEnvironment().toDataSet(
+                env.getBatchTableEnvironment().fromValues(
+                        DataTypes.ROW(DataTypes.FIELD("name", DataTypes.STRING()),
+                                DataTypes.FIELD("age", DataTypes.INT())),
+                        Arrays.stream(NAME_ARRAY).map(n -> Row.of(n, random.nextInt())).collect(Collectors.toList())), Row.class);
```
Comment (ruanwenjun) [inline seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-fake/src/main/java/org/apache/seatunnel/flink/source/FakeSource.java]: Use `random.nextInt(200)` is better, `random.nextInt()` might return negative value, this is not related to age.

**Review by ruanwenjun — CHANGES_REQUESTED** [review]

File: seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-fake/src/main/java/org/apache/seatunnel/flink/source/FakeSource.java (hunk 1 above)
Comment (Hisoka-X) [reply seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-fake/src/main/java/org/apache/seatunnel/flink/source/FakeSource.java]: Thanks for advise

**Review by ruanwenjun — APPROVED** [review]
Add `FakeSource` looks good to me, but I am not clear if we need to add `limit` parameter, since no one seems to set this parameter in batch job.

**Comment by Hisoka-X** [comment]
> Add `FakeSource` looks good to me, but I am not clear if we need to add `limit` parameter, since no one seems to set this parameter in batch job.

It's more like top N in user debug mode

**Review by CalvinKirs — APPROVED** [review]
LGTM
